README.md

```diff
@@ -487,3 +487,39 @@ USER_WIDGETS=ConnectTerminalWidget;ServerWidget
 Use the file `TemplateUserWidget.py` for more practical information and to start your own widgets!
 
 
+## Using LabGUI Core Widgets ##
+
+Built into LabGUI, there are a set of core widgets in which it bases it's operation.
+Among these are several designed to help analyse/manipulate data and aid in seemless communication with devices.
+
+These widgets can be enabled/disabled from the `Window` dropdown menu.
+
+#### Remote Enable  Widget ####
+This widget allows the user to bring any device into a specific REN state, as well as set the default state for all connected devices.
+
+#### Instrument Widget ####
+This widget allows the user to select which instruments to connect to, from a list of device drivers as well as connected ports. By default, the state of this widget is loaded on launch, and saved on exit.
+
+#### Command Widget ####
+This widget is the GPIB Command-Line Widget, described above.
+
+#### Output File Widget ####
+This widget allows to the user to specify where data is saved, as well as leave some notes as comments in that datafile. Notes left by the user is saved/loaded by default.
+
+#### Script Widget ####
+This widget allows the user to specify which script is to be run via DataTaker.
+
+#### Properties Widget ####
+This widget allows the user to select from a list of connected devices, and modify any properties that may be available within the devices' driver.
+
+#### Function Widget ####
+This widget allows the user to select from a list of connected devices, and run any functions that may be available within the devices' driver.
+
+#### User Data Widget ####
+This widget is an extension to the Output File Widget comment saver, 
```

Comment:
I initially designed it so that one can provide variables in the script. And change their values live

Reply:
This widget is oriented more for readability of data files. As such, it will save certain 'variables' with values, which can be manually changed by the user, for managing datafiles afterwards. This may be especially useful if the user is planning on pragmatically analyzing their data independently, as the variables are set in the header in a way which can be easily read by using the LabGUI library (specifically the function IOTool.load_file_windows, where it is retrievable as a tuple under the second return variable, label. If someone analyzing their data were to user IOTool.load_file_windows to load multiple datafiles, it could then be easily organized by certain assigned user data value(s).

Reply:
Please refer to the latest commit for added documentation on this widget